### PR TITLE
Python 3 changes plus automated testing

### PR DESCRIPTION
**Pull request by cclauss**
This seems to be blocking Python 3 progress at https://github.com/nodejs/node/pull/29451#issuecomment-533727436

Test results: https://travis-ci.com/cclauss/tap2junit

@jbergstroem Your review please.  Will require a new release to PyPI #8

@sam-github @refack @bnoordhuis @Trott Your reviews too please.

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -10,4 +10,3 @@ junit-xml = "*"
 [dev-packages]
 
 [requires]
-python_version = "2.7"
```
Comment (MattIPv4) [inline Pipfile]: Should this not be "2.7,3.7", to reflect what's being used in the Travis config?

Comment (cclauss) [reply Pipfile]: I believe that leaving it out is in alignment with the docs...
See https://pipenv-fork.readthedocs.io/en/latest/basics.html#specifying-versions-of-python

Comment (MattIPv4) [reply Pipfile]: Ah, sure :)

File: tap2junit/__main__.py [hunk 2]
```diff
@@ -2,7 +2,7 @@
 import os
 import platform
 from junit_xml import TestSuite, TestCase
-from tap13 import TAP13 as tap13
+from .tap13 import TAP13 as tap13
```
Comment (sam-github) [inline tap2junit/__main__.py]: just to check, this work on py2 and py3?

**Comment by sam-github** [comment]
As long as this continues to work on py2 and py3, it LGTM.

Who has credentials to land it?

@cclauss once landed, do you have pypi publish credentials now?

File: tap2junit/__main__.py (hunk 2 above)
Comment (cclauss) [reply tap2junit/__main__.py]: Thanks,  Changed to absolute import.

**Comment by cclauss** [comment]
Only @jbergstroem can land it and only @jbergstroem has the PyPI credentials.

**Comment by MattIPv4** [comment]
Alternative plan if this isn't landed soon, as this a blocker?
Vendor in a fork of this with our fixes where it is needed?

**Comment by cclauss** [comment]
See: nodejs/build#1921 for all the places in ansible scripts where it is installed.  I kinda hate vendoring in.

**Comment by MattIPv4** [comment]
We could also publish the fork on PyPi under `tap2junit-node` or similar?

**Comment by sam-github** [comment]
@jbergstroem In the interests of minimizing maintenance burden for you, if you are willing, you could just add either @cclauss , or me (@sam-github), or any other Node.js TSC member to admin on this repo, I think that would allow us to do the maintenance.

But if you have time to add cclauss to pypi and transfer it yourself, that works great, too!

I'm not fan of ansible, and would be happy to vendor it, but I'm not keen to grout out all the references to it in our ansible scripts. The low-maintenance fix is definitely to transfer it, if Johan is willing.

**Comment by sam-github** [comment]
@cclauss I think you can land this, can you try?

**Pull request merged**